internal/authz/authz.go

```diff
@@ -5,44 +5,46 @@ import (
 	"net/http"
 
 	"github.com/go-logr/logr"
+
 	"github.com/kubeshop/kusk-gateway/internal/cloudentity"
 )
 
 type AuthorizationServer struct {
 	log logr.Logger
 }
 
-func (a *AuthorizationServer) check(w http.ResponseWriter, r *http.Request) {
-	url := r.Header.Get(cloudentity.HeaderAuthorizerURL)
+func (a *AuthorizationServer) check(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Not sure changing `w` and `r` to `writer` and `request` is adding anything
`w` and `r` is idiomatic

Reply:
the linter was complaininig.

Reply:
Really? What was it saying?

Reply:
Hmm okay fair enough